README.md

```diff
@@ -90,12 +49,11 @@ Steps:
 * G - R Threshold is set to 5 for normal situation.
```

Comment:
The threshold  G - R is set to 2 for now

Reply:
Change made